glean-core/src/traits/mod.rs

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The different metric types supported by the Glean SDK to handle data.
```

Comment:
Can we call out that they are considered `unstable` / `experimental` right now?
I have some ideas how we might improve them to better accomodate for a nice Rust-y API, so I wouldn't want us to have them be considered stable yet (and yes, I know, no one should be using glean-core, but this also acts as docs for us)

(In _theory_ we should hide them behind a feature flag, but that will make integration in this repository with the RLB harder)

Reply:
Sure! Let me flag these as unstable / experimental here.